### DISCUSSION TITLE
Fail to generate

### DISCUSSION THREAD

**Discussion by isasenoaji** [spatie/browsershot#522] [Q&A]
Hi, im using linux ubuntu 16.xx. when i tried to save using this library i got this message "For some reason Chrome did not write a file at `/tmp/1797499489-0804132001624430435/screenshot.png`. Error output: `[0623/134035.886076:ERROR:gpu_init.cc(441)] Passthrough is not supported, GL is swiftshader"

anybody can help me?

**Comment by condor-bird** [comment]
It is necessary that the file has already been created before writing to it. The save () method itself does not create a file at the specified path if the file does not exist.
